sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/DirectoryApiTests.java

```diff
@@ -585,6 +601,46 @@ public void deleteDirDoesNotExistAnymore() {
         assertEquals(BlobErrorCode.BLOB_NOT_FOUND.toString(), e.getErrorCode());
     }
 
+    @Disabled("Requires manual OAuth setup and creates 5000+ files")
+    @DisabledIf("olderThan20230803ServiceVersion")
+    @Test
+    public void deletePaginatedDirectory() {
```

Comment:
Have we been able to successfully manually run this test?

Reply:
Yes, confirmed that it continues to delete the directory successfully with a response code of 200, and confirmed the continuation token passed in the request and response headers.